context.md

```diff
@@ -8,20 +8,73 @@
 - Use bind to create a new method bound to an object context
 - Use apply/call to execute a method in a different context
 
+## TLDR:
+
+`this` is whatever was to the **left of the period** when it was called, unless:
+- You're in an event listener function, in which case `this` is the thing that was clicked on
+- You're in another callback function, in which case `this` is probably the `Window`
+
+### For example
```

Comment:
Before asking them to sift through a larger example, maybe show them very short simple examples of each of the three examples (object, event listener, callback).

- Adrian's Mom